packages/cursorless-engine/src/actions/FollowLink.ts

```diff
@@ -10,8 +10,13 @@ import { Actions } from "./Actions";
 import { SimpleAction, ActionReturnValue } from "./actions.types";
 
 export default class FollowLink implements SimpleAction {
-  constructor(private actions: Actions) {
+  openInSplit = false;
+  constructor(
+    private actions: Actions,
+    private options: { openInSplit: boolean },
+  ) {
     this.run = this.run.bind(this);
+    this.options = options;
```

Comment:
When you use private on a constructor argument you don't need to assign it yourself; that is done automatically
Just remove lines `openInSplit = false;` and `this.options = options;`